packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -22,7 +22,7 @@ class BuildIOSCommand extends BuildSubCommand {
   BuildIOSCommand({ @required bool verboseHelp }) {
     addTreeShakeIconsFlag();
     addSplitDebugInfoOption();
-    addBuildModeFlags(defaultToRelease: false);
+    addBuildModeFlags(defaultToRelease: true);
```

Comment:
This is a big behavior change, who knows what build scripts out there are leaving off `--debug`.  Can we leave the default behavior and just fix up the text?

Reply:
This doesn't change the default, see https://github.com/flutter/flutter/pull/56720/files#diff-c616f0a92b2c8f4d7b8d95d2367e0ff6R61

Unless a user passed simulator, we're defaulting to release anyway.

Reply:
Doesn't it have the side effect of setting _defaultBuildMode?
https://github.com/flutter/flutter/blob/0a4f6cdef0c39c73cab310342784813dfe6b5ec8/packages/flutter_tools/lib/src/runner/flutter_command.dart#L381

Reply:
But that is overriden again on L61

Reply:
So it is.

Reply:
Its a confusing API